libcxx/test/libcxx/memory/indirect/indirect.obs/deref.nodiscard.verify.cpp

```diff
@@ -0,0 +1,25 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// REQUIRES: std-at-least-c++26
+
+// <memory>
+
```

Comment:
Please put all the nodiscard checks in a single file. Separating them like this is really not necessary.